frontend/client/styles/screens/manage_teams.scss

```diff
@@ -12,11 +17,25 @@
   cursor: pointer;
 }
 
+.mobile-msg {
+  display: none;
```

Comment:
Nitpick here:

Let's make this
```scss
@include media('>phone') {
    display: none;
}
```

As a general rule I think its much easier to read this conditional scss code when it's explicit about what's for mobile and what's for desktop. I've found when I don't write the code this way, I commonly start mixing the paradigm up (i.e. some classes the default is mobile, while others the default is desktop), and code integrity degrades over time.

Reply:
Fixed, no prob - makes sense